Resources/config/services.yml

```
@@ -29,6 +30,16 @@ parameters:
         support: "https://support.ez.no"
 
 services:
+    # EventSubscriber
+
+    support_tools.promote_platform.response_subscriber:
+        class: EzSystems\EzSupportToolsBundle\EventSubscriber\AddXPoweredByHeader
```

Could you use FQCN as a service name instead?
```suggestion
    EzSystems\EzSupportToolsBundle\EventSubscriber\AddXPoweredByHeader:
```